src/UserInterfaceServiceProvider.php

```diff
@@ -247,6 +247,7 @@ private function registerBladeComponents(): void
         Blade::component('ark::loader-icon', 'ark-loader-icon');
         Blade::component('ark::message', 'ark-message');
         Blade::component('ark::metadata', 'ark-metadata');
+        Blade::component('ark::metadata-title', 'ark-metadata-title');
```

Comment:
there is already a metadata-tags component that sets metadata data, why add a new component instead of adjusting the existing one? 🤔

Reply:
@ItsANameToo yeah I can reuse the same component, will fix in a moment